CriteoPublisherSdk/Tests/UnitTests/Configuration/CR_IntegrationRegistryTests.m

```diff
@@ -143,4 +143,35 @@ - (void)testIsAdMobMediationPresent_GivenAdMobMediationNotInRuntime_ReturnNo {
   XCTAssertFalse(isPresent);
 }
 
-@end
\ No newline at end of file
+- (void)testFilterAdUnits_GivenProfileIdNotGAM_RemoveRewardedAdUnit {
```

Comment:
Do you mind adding a test case in InHouse functional tests ? It a little bit a duplication of your test here, but it's to keep a layer of functional tests that shows the features and boundary of the SDK